pcsx2-qt/Settings/GraphicsSettingsWidget.cpp

```diff
@@ -810,6 +810,7 @@ void GraphicsSettingsWidget::onCaptureContainerChanged()
 
 	m_ui.videoCaptureCodec->disconnect();
 	m_ui.videoCaptureCodec->clear();
+	//: This string possibly refers to default audio/video codecs.
```

Comment:
Yeah.  If you want to separate them, Qt should have a way to add disambiguation comments here as well, I think it's something like `tr("Default", "Comment")`